Disable command "pnpm self-update" as we install via apk

diff --git a/dist/pnpm.cjs b/dist/pnpm.cjs
index b93ea78..8da50c3 100644
--- a/dist/pnpm.cjs
+++ b/dist/pnpm.cjs
@@ -220258,6 +220258,7 @@ var require_selfUpdate = __commonJS({
       });
     }
     async function handler(opts, params) {
+      throw new error_1.PnpmError("CANT_SELF_UPDATE_IN_APK", "You should update pnpm with apk");
       if ((0, cli_meta_1.isExecutedByCorepack)()) {
         throw new error_1.PnpmError("CANT_SELF_UPDATE_IN_COREPACK", "You should update pnpm with corepack");
       }
